src/librustc_mir/transform/check_consts/ops.rs

```diff
@@ -353,16 +353,16 @@ impl NonConstOp for StaticAccess {
             item.tcx.sess,
             span,
             E0013,
-            "{}s cannot refer to statics, use \
-                                        a constant instead",
+            "{}s cannot refer to statics",
             item.const_kind()
         );
+        err.help("consider extracting the value of the static to a const, and referring to that");
         if item.tcx.sess.teach(&err.get_code().unwrap()) {
             err.note(
                 "Static and const variables can refer to other const variables. \
                     But a const variable cannot refer to a static variable.",
```

Comment:
```suggestion
                    a `const` variable cannot however refer to a static variable.",
```
(+ split this sentence into a second `.note("...")`)

Reply:
There seems to be a different style for `--teach` diagnostics, in which multi-line notes are acceptable.

Reply:
ah 👍